### PR TITLE
Disregard sectiondef boundaries when de-duplicating adjacent memberde…

### PR DESCRIPTION
**Pull request by evanlenz**
…f briefdescriptions (like "Constructor"). Addresses boostorg/json#440

### CODE REVIEW

File: include/docca/base-stage1.xsl [hunk 1]
```diff
@@ -171,7 +171,7 @@
   </xsl:template>
 
           <xsl:template mode="overload-list" match="memberdef">
-            <xsl:apply-templates select="briefdescription[not(. = ../preceding-sibling::*/briefdescription)]"/>
+            <xsl:apply-templates select="briefdescription[not(. eq preceding::briefdescription[parent::memberdef][1])]"/>
```
Comment (vinniefalco) [inline include/docca/base-stage1.xsl:174]: LOL That's it?!

**Comment by vinniefalco** [comment]
Does the example build for you, with "b2 example" ?

**Comment by evanlenz** [comment]
I'm getting this:

evanl@THINKPAD-T480 ~/boost/tools/docca/example
$ ../../../b2.exe
Jamfile:33: in modules.load
*** argument error
* rule glob ( wildcards + : excludes * )
* called with: (  )
* missing argument wildcards
C:/cygwin64/home/evanl/boost/tools/build/src/build\project.jam:1274:see
definition of rule 'glob' being called
C:/cygwin64/home/evanl/boost/tools/build/src/build\project.jam:372: in
load-jamfile
C:/cygwin64/home/evanl/boost/tools/build/src/build\project.jam:64: in load
C:/cygwin64/home/evanl/boost/tools/build/src/build\project.jam:142: in
project.find
C:/cygwin64/home/evanl/boost/tools/build/src\build-system.jam:618: in load
C:/cygwin64/home/evanl/boost/tools/build/src/kernel\modules.jam:295: in
import
C:/cygwin64/home/evanl/boost/tools/build/src/kernel/bootstrap.jam:139: in
boost-build
C:/cygwin64/home/evanl/boost/boost-build.jam:17: in module scope

**Comment by vinniefalco** [comment]
I am investigating...

File: include/docca/base-stage1.xsl (hunk 1 above)
Comment (evanlenz) [reply include/docca/base-stage1.xsl:174]: Yeah, although the `preceding` axis can be slow. I'm going to check in a better implementation shortly.

**Comment by evanlenz** [comment]
Happily, this code change should also fix boostorg/json#387, because now it is displaying the briefdescription if it has a different value than the immediately preceding briefdescription (not a different value than *any* of the previous briefdescriptions).

**Pull request merged**